warehouse/translations/en/LC_MESSAGES/warehouse.po

```diff
@@ -18,50 +18,10 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 1.3\n"
 
-#: warehouse/templates/packaging/detail.html:87
```

Comment:
I'm not sure what happened here either, rerunning `make extract-translations && make compile-translations` seems to have regenerated the strings and gives a saner diff.